apps/fluent-tester/docs/android.md

```diff
@@ -43,7 +43,7 @@ adb reverse tcp:8081 tcp:8081
 yarn start
 ```
 
-- APKs: The emulator is running on arm64 so any APKs that need to be installed need to be for arm64. Here is a [link](https://www.apkmirror.com/apk/google-inc/android-accessibility-suite/android-accessibility-suite-13-1-0-501229322-release/android-accessibility-suite-13-1-0-501229322-2-android-apk-download/) to the accessibility APK which includes TalkBack (voiceover)
```

Comment:
Thanks!

Reply:
Hmm neither of these are working :/

Reply:
Also, pretty sure I should be able to test this locally without having to push commits but running yarn test-links just hangs for me

Reply:
I just realized.. we shouldn't really have a link to apkmirror IMO, that's not an official Android website. Can we just ignore or remove it?